File: ClinicSystem/src/com/example/clinicsystem/models/classes/M_Drug.java
package com.example.clinicsystem.models.classes;

import com.example.clinicsystem.helpers.DatabaseConnection;
import com.example.clinicsystem.models.enums.EquipmentType;

public class M_Drug extends M_Equipment {
    private String expiryDate, productionDate;

    public M_Drug(String name, EquipmentType equipmentType, int price, int id, int quantity, boolean isAvailable, String expiryDate, String productionDate) {
        super(name, equipmentType, price, id, quantity, isAvailable);
        this.expiryDate = expiryDate;
        this.productionDate = productionDate;
    }

    public M_Drug() {
    }

    public int addDrug() {
        DatabaseConnection databaseConnection = DatabaseConnection.getINSTANCE();

        return databaseConnection.insertInto("drug", "(Equipment_ID, Drug_ExpiryDate, Drug_ProductionDate)",
                "('" + this.getId() + "', '" + this.expiryDate + "', '" + this.productionDate + "')");
    }

    public String getExpiryDate() {
        return expiryDate;
    }

    public void setExpiryDate(String expiryDate) {
        this.expiryDate = expiryDate;
    }

    public String getProductionDate() {
        return productionDate;
    }

    public void setProductionDate(String productionDate) {
        this.productionDate = productionDate;
    }
}

File: clinic/M_Drug.php
<?php
include_once "M_Equipment.php";

class M_Drug extends M_Equipment {
    private $expiryDate, $productionDate;

    public function __construct($name, $equipmentType, $price, $id, $quantity, $isAvailable, $expiryDate, $productionDate) {
        $this->name =          $name;
        $this->equipmentType = $equipmentType;
        $this->price =         $price;
        $this->id =            $id;
        $this->quantity =      $quantity;
        $this->isAvailable =   $isAvailable;
        $this->expiryDate =    $expiryDate;
        $this->productionDate =$productionDate;
    }

    public function addDrug() {
		$conn = DatabaseConnection::getINSTANCE();
		
		return $conn->insertInto("drug", "(Equipment_ID, Drug_ExpiryDate, Drug_ProductionDate)", "('". 
		$this->id . "', '" . $this->expiryDate . "', '" . $this->productionDate . "')");
    }

    public function getExpiryDate() {
        return $this->expiryDate;
    }

    public function setExpiryDate($expiryDate) {
        $this->expiryDate = $expiryDate;
    }

    public function getProductionDate() {
        return $this->productionDate;
    }

    public function setProductionDate($productionDate) {
        $this->productionDate = $productionDate;
    }
}
?>